presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -676,6 +677,19 @@ public HiveClientConfig setOrcBloomFiltersEnabled(boolean orcBloomFiltersEnabled
         return this;
     }
 
+    @NotNull
+    public Double getOrcBloomFilterFpp()
+    {
+        return orcBloomFilterFpp;
+    }
+
+    @Config("hive.orc.bloom-filter-fpp")
```

Comment:
Maybe this should be the `default-bloom-filter-fpp` or `bloom-filter-fpp-default` since the user can override it? Though for table format we just say `storage-format`, so this would be consistent.